support/cas-server-support-pm-webflow/src/test/java/org/apereo/cas/pm/web/flow/actions/ValidatePasswordResetTokenActionTests.java

```diff
@@ -31,6 +39,24 @@ void verifyInvalidTicket() throws Throwable {
         assertEquals(CasWebflowConstants.TRANSITION_ID_INVALID_PASSWORD_RESET_TOKEN, validatePasswordResetTokenAction.execute(context).getId());
     }
 
+    @Test
+    void verifyGoodTicket() throws Throwable {
+        ClientInfoHolder.setClientInfo(new ClientInfo());
+        val id = UUID.randomUUID().toString();
+        val ticket = new TransientSessionTicketImpl();
+        ticket.setId(id);
+        ticket.setCreationTime(ZonedDateTime.now(ZoneId.systemDefault()));
```

Comment:
System-Default cannot be zone. It will different for every run in CI for example, depending on where it runs.

Reply:
I will use `ZoneOffset.UTC` instead as seen in other tests.